src/main/java/domain/Position.java

```diff
@@ -0,0 +1,46 @@
+package domain;
+
+import java.util.Objects;
+
+public class Position implements Comparable<Position> {
+
+    private int position;
+
+    public Position() {
+        this.position = 0;
```

Comment:
주 생성자를 호출하는 방식으로 바꿔보면 어떨까요?
```suggestion
        this(0);
```